tests/strategies/energy_parameters/test_heat_pump.py

```diff
@@ -140,16 +140,18 @@ def test_event_traded_energy_updates_temp_increase(energy_params):
         energy_params.event_market_cycle(CURRENT_MARKET_SLOT)
         traded_energy = 0.1
         energy_params.event_traded_energy(CURRENT_MARKET_SLOT, traded_energy)
+        # energy_params._update_last_time_slot_data(CURRENT_MARKET_SLOT)
```

Comment:
This can be removed now.
IMO the test method name should also be renamed, for example: `test_event_market_cycle_updates_temp_increase_if_energy_was_traded`

Reply:
Fixed both and thanks!